tests/service/test_service_keytab.yml

```diff
@@ -33,7 +33,7 @@
       - user02
       action: member
     register: result
-    failed_when: not result.changed
+    failed_when: not result.changed or result.failed
```

Comment:
This change is unrelated.

Reply:
This change is in a TEMP commit, so service tests pass.